ckanext/budgetviz/groupbarchart_munc/theme/public/custom.css

```diff
@@ -15,6 +15,9 @@
 }
 
 .chartarea {
-    margin-top: 20px;
+    padding: 10px 25px;
```

Comment:
Yes. Though 13 inch laptops can have high resolution. So speaking more specifically, the viz dont work for devices below res that are lower than 980.